db.go

```diff
@@ -33,32 +34,54 @@ type Stat struct {
 }
 
 func Open(options Options) (*DB, error) {
+	// banner
+
+	// logger
+	logPath := path.Join(DefaultLogDir, time.Now().Format("2006-01-02")+"_"+DefaultLogFileName)
+	err := NewJSONLogger(
+		WithInfoLevel(),
+		WithFileRotationPath(logPath),
+		WithDisableConsole(),
+		WithTimeLayout("2006-01-02 15:04:05"),
+		WithEnableHighlighting(),
+	)
+	if err != nil {
+		Error("open db failed", zap.Error(err))
+	}
+	defer Sync()
```

Comment:
the sync will be called in `Sync` and `Close` function?

Reply:
yeah, i will fix it